src/pages/cart/Cartcount.js

```diff
@@ -1,31 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Cartcount.scss';
-
-const Cartcount = ({ value, onRemove, plusCount, minusCount }) => {
+const Cartcount = ({ value, onRemove }) => {
+  const [count, setCount] = useState(0);
+  const countBtn = e => {
+    if (e.target.id === 1) {
+      setCount(value.quantity - 1);
+    } else {
+      setCount(value.quantity + 1);
+    }
+  };
   return (
     <tbody>
       <tr className="cartItemList">
         <th className="cartItemImg">
-          <img className="imgSize" src={value.thumbnail_image} alt="meat" />
+          <img className="imgSize" src={value.thumbnail} alt="meat" />
         </th>
         <th className="cartItemName">
-          <p className="cartItemNameTitle">{value.name}</p>
-          <span className="cartItemOption"> {value.option}</span>
-        </th>
-        <th className="cartItemWeight">
-          {(value.base_unit * value.count).toLocaleString()}g 기준
+          <p className="cartItemNameTitle">{value.productName}</p>
+          <span className="cartItemOption"> {value.productOptionName}</span>
         </th>
+        <th className="cartItemWeight">{value.baseUnit} 기준</th>
         <th className="cartItemCount">
-          <button className="clickMinusButton" onClick={minusCount}>
+          <button className="clickMinusButton" id="1" onClick={countBtn}>
```

Comment:
id를 1, 2 같은 값으로 주는 것은 다른 부분들과 충돌이 발생할 수 있으니
```suggestion
          <button className="clickMinusButton" onClick={() => countBtn(1)}> // 혹은 -1
``
차라리 이런 식으로 증감값을 넘겨주는 건 어떨까요?